### PR TITLE
pvs/v1037 ex_docmd.c

### PR DESCRIPTION
**Pull request by ngortheone**
https://www.viva64.com/en/w/v1037/

### CODE REVIEW

File: src/nvim/ex_docmd.c [hunk 1]
```diff
@@ -1156,13 +1156,24 @@ static void get_wincmd_addr_type(char_u *arg, exarg_T *eap)
...
+    case 't':
+    case Ctrl_T:
+    case 'b':
+    case Ctrl_B:
+    case 'p':
+    case Ctrl_P:
+    case '=':
+    case CAR:
+      // window size or any count
       eap->addr_type = ADDR_LINES;
```
Comment (ngortheone) [inline src/nvim/ex_docmd.c]: `#define ADDR_LINES              0`

Comment (justinmk) [reply src/nvim/ex_docmd.c]: This is different semantically that "no count", which we've now lost as a hint/documentation.

This particular PVS warning is really annoying. I wish we could find a way to disable some warnings project-wide.

In this case I don't think this is a good change.

**Review by justinmk — CHANGES_REQUESTED** [review]
mark as ignored (`// V1037`) or disable V1037 in pvscheck.sh ?

File: src/nvim/ex_docmd.c (hunk 1 above)
Comment (ngortheone) [reply src/nvim/ex_docmd.c]: ack. will add ignore to this one.

**Comment by ngortheone** [comment]
Looks like flaky tests.. Change consists of comments only.

**Comment by ngortheone** [comment]
@blueyed @justinmk can you restart ci? looks like everything flaked-out :)

**Comment by blueyed** [comment]
@ngortheone 
Builds are a bit flaky.

As for comment-only changes we could use `[ci skip]`, also when (squash)merging.

**Pull request merged**